offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/AppViewModel.groovy

```diff
@@ -1,61 +1,42 @@
 package life.qbic.portal.offermanager.components
 
 
-import life.qbic.datamodel.dtos.business.Affiliation
-import life.qbic.portal.offermanager.dataresources.persons.AffiliationResourcesService
 import life.qbic.portal.offermanager.security.Role
-import life.qbic.portal.offermanager.security.RoleService
-
 
 /**
  * A simple DTO for data displayed in the PortletView
  *
  * This class holds information and data to be displayed in the view.
  * It can contain JavaBean objects to enable views to listen to changes in the values.
  *
- * @since: 1.0.0
- * @author: Tobias Koch
+ * @since 1.0.0
+ * @author Tobias Koch
  */
 class AppViewModel {
     final ObservableList successNotifications
     final ObservableList failureNotifications
 
-    private final AffiliationResourcesService service
-
     private final Role role
 
     boolean createOfferFeatureEnabled
-
     boolean createCustomerFeatureEnabled
-
     boolean searchCustomerFeatureEnabled
-
     boolean maintainProductsFeatureEnabled
 
-    AppViewModel(AffiliationResourcesService service,
-                 Role role) {
-        this(new ArrayList<Affiliation>(),
-                new ArrayList<String>(),
-                new ArrayList<String>(),
-                new ArrayList<String>(),
-                service,
-                role)
+    AppViewModel(Role role) {
+        this(new ArrayList<String>(), new ArrayList<String>(), role)
     }
 
-    private AppViewModel(List<Affiliation> affiliations,
-                         List<String> academicTitles,
-                         List<String> successNotifications,
+    private AppViewModel(List<String> successNotifications,
                          List<String> failureNotifications,
-                         AffiliationResourcesService service,
                          Role role) {
```

Comment:
why did you change the constructors here?

Reply:
I changed it because the service is a relict of long-forgotten times and not needed in the AppView.

Reply:
So are the affiliations and academicTitles.